templates/uuid_fact.json.erb

```diff
@@ -0,0 +1,2 @@
+{"smart_proxy_uuid": "<%= scope.lookupvar("foreman_proxy::instance_id") %>",
+"smart_proxy_uuid_signature": "<%= @uuid_fact_signature %>"}
```

Comment:
I thin it would be good to add to the acceptance tests a test of this file as that will help ensure things like idempotency.

Reply:
Do you insist on having an explicit test for it? The acceptance tests we have will fail like this https://github.com/theforeman/puppet-foreman/pull/914/checks?check_run_id=2351829843#step:4:6893 if the instance uuid gets changed